### PR TITLE
Addon-controls: Add reset UI to ArgsTable

### PR DESCRIPTION
**Pull request by shilman**
Issue: #11487 

## What I did

- [x] ~~Fix a critical bug where args were not being properly updated on the `Docs` tab~~ => #11552
- [x] Add reset button to `ArgsTable` in both addon panel & `Docs` tab

## What I need

- [x] @domyen any chance you can style this?

## How to test

```
cd examples/official-storybook
yarn storybook
open http://localhost:9011/?path=/docs/addons-docs-props--controls
```

View in both addon panel and `Docs` tab

### CODE REVIEW

File: addons/docs/src/blocks/Props.tsx [hunk 1]
```diff
@@ -63,7 +66,8 @@ const useArgs = (storyId: string, storyStore: StoryStore): [Args, (args: Args) =
   const updateArgs = useCallback((newArgs) => storyStore.updateStoryArgs(storyId, newArgs), [
     storyId,
   ]);
-  return [args, updateArgs];
+  const resetArgs = useCallback(() => storyStore.updateStoryArgs(storyId, initialArgs), [storyId]);
```
Comment (tmeasday) [inline addons/docs/src/blocks/Props.tsx:69]: This doesn't seem right:

```suggestion
  const resetArgs = useCallback((argNames?: string[]) => storyStore.resetStoryArgs(storyId, argNames), [storyId]);
```

**Review by tmeasday — APPROVED** [review]

**Comment by ndelangen** [comment]
Can we add a story for this?

Screenshot 2020-07-15 at 11 38 35

**Comment by shilman** [comment]
@ndelangen added, i think

**Comment by domyen** [comment]
Thanks for updating the stories :)

Styled the buttons like so:
image

**Comment by shilman** [comment]
Looks great @domyen 💯

**Pull request merged**